CHANGELOG.md

```diff
@@ -1,12 +1,15 @@
 # Vx.x.x (Pre-release)
 
 ## Release Highlights
+- 🕵️‍♀️ Vulnerabilities have been addressed
```

Comment:
Please remove this line. The release highlights are usually written by the project maintainers. Right before a release is done.

Reply:
Thanks. Just removed it.